mteb/model_meta.py

```diff
@@ -70,11 +70,12 @@ class ModelMeta(BaseModel):
         reference: A URL to the model's page on huggingface or another source.
         languages: The languages the model is intended for specified as a 3 letter language code followed by a script code e.g. "eng-Latn" for English
             in the Latin script.
-        use_instuctions: Whether the model uses instructions E.g. for prompt-based models. This also include models that require a specific format for
+        use_instructions: Whether the model uses instructions E.g. for prompt-based models. This also include models that require a specific format for
             input such as "query: {document}" or "passage: {document}".
         zero_shot_benchmarks: A list of benchmarks on which the model has been evaluated in a zero-shot setting. By default we assume that all models
             are evaluated non-zero-shot unless specified otherwise.
     """
+    model_config = ConfigDict(extra="forbid")
```

Comment:
For now, I added `extra="forbid"` to make sure that all parameters of model metadata are correct, but I can remove it